pkg/controller/loadtest.go

```diff
@@ -36,6 +35,7 @@ const (
 	controllerAgentName = "kangal"
 	falseString         = "false"
 	trueString          = "true"
+	syncHandlerTimeout  = time.Minute
```

Comment:
Not needed to make this configurable?

Reply:
No; if sync takes more than one minute then something is very wrong, and increasing it would not be a good fix.